runtime/drivers/duckdb/catalog.go

```diff
@@ -46,7 +46,7 @@ func (c *connection) findEntries(ctx context.Context, whereClause string, args .
 		var objBlob []byte
 		e := &drivers.CatalogEntry{}
 
-		err := rows.Scan(&e.Name, &e.Type, &objBlob, &e.Path, &e.SizeInBytes, &e.Embedded, &e.CreatedOn, &e.UpdatedOn, &e.RefreshedOn)
+		err := rows.Scan(&e.Name, &e.Type, &objBlob, &e.Path, &e.BytesIngested, &e.Embedded, &e.CreatedOn, &e.UpdatedOn, &e.RefreshedOn)
```

Comment:
This is still called `size_in_bytes` in the DuckDB migrations – need to change to `bytes_ingested` there as well